src/lib/AnnotationControlsFSM.ts

```diff
@@ -53,7 +54,7 @@ export default class AnnotationControlsFSM {
             return stateModeMap[this.currentState];
         }
 
-        if (input === AnnotationInput.RESET) {
+        if (input === AnnotationInput.RESET || input === AnnotationInput.INIT) {
```

Comment:
Should we go to NONE state if we're in explicit region/highlight mode and the input is `INIT`?

Reply:
I'm not sure - my guess is we still want to stay in explicit region / highlight mode.

Reply:
We probably only want to go to NONE state only when we are not in explicit region / highlight mode.

Reply:
I agree, should be under
```
case AnnotationState.HIGHLIGHT_TEMP:
case AnnotationState. REGION_TEMP:
```

Reply:
So something like this:
```
if (input === AnnotationInput.INIT && this.currentState === AnnotationState.NONE) {
    this.currentState = AnnotationState.NONE;
    return stateModeMap[this.currentState];
}
```